conf/messages

```diff
@@ -49,19 +49,19 @@ capture-sa-utr.details.line_1                               = Your UTR helps us
 capture-sa-utr.details.line_2                               = I cannot find my UTR
 capture-sa-utr.details.line_3                               = My partnership does not have a UTR
 capture-sa-utr.link_1                                       = I cannot find my UTR
-capture-sa-utr.error                                        = Enter a Unique Taxpayer Reference in the correct format
+capture-sa-utr.error                                        = Enter the partnership’s Unique Taxpayer Reference in the correct format
 
 ## Capture PostCode Page
-capture-postcode.title                                      = What is the postcode where the partnership is registered for Self Assessment?
-capture-postcode.heading                                    = What is the postcode where the partnership is registered for Self Assessment?
+capture-postcode.title                                      = What is the postcode used to register the partnership for Self Assessment?
+capture-postcode.heading                                    = What is the postcode used to register the partnership for Self Assessment?
 capture-postcode.hint_text                                  = For example, AB1 2YZ
 capture-postcode.error.not_entered                          = Enter the postcode where the partnership is registered for Self Assessment
```

Comment:
The spreadsheet has "Enter the postcode where the business is registered for Self Assessment" which seems unusual. What is the correct text?

Reply:
It looks like this error message update has been removed from later versions of the update spreadsheet.